beginend.el

```diff
@@ -49,18 +49,28 @@
   (re-search-backward "[^[:space:]]" nil t)
   (forward-char))
 
+(defun beginend--out-of-bounds-p (point)
+  "Return non-nil if POINT is outside [`point-min', `point-max'].
+This is possible if buffer was narrowed after POINT was stored."
+  (not (<= (point-min) point (point-max))))
```

Comment:
Oh, okey.  Somehow I was under the impression it's a 25+ feature.  Let's go ahead with this then.  But we should probably add a version constraint to the requirements.